### DISCUSSION TITLE
Getting Zero File Size When Converting to FLAC

### DISCUSSION THREAD

**Discussion by ThunderBoyDavid** [enzo1982/freac#466] [Help & Support]
i have several folders with 100 to 200 WAV files in them. i wanted to convert the WAVs to FLAC. Don't ask.. i just wanted too. i copied one folder at a time and started the encode process. Most of the files did fine... But there were around 18 or so that had a Zero File Size. i went back and pulled those files and re-ran them and that time they did fine. Is there a trick to making all the files do right the 1st time?
Is there something i should be looking for that might be wrong with the files?? Please Help!!?? Thanks, David, ThunderBoyDavid

fre:ac v1.1.7

Every Fear Hides A Wish

**Comment by enzo1982** [comment]
Thank you for reporting this!

If the files converted fine when you tried again, there is nothing wrong with the source files.

I haven't seen such an issue yet. Did you maybe look at the folder/files while the conversion was still running? On Windows, files that have not finished converting will display with a zero file size. Once the conversion is finished, the file sizes will usually update automatically in Explorer, but in some cases or when using a different file manager, it may be necessary to update the view (e.g. by pressing F5).
